code/src/components/CheckboxStyled.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+
+const HiddenCheckbox = styled.input.attrs({ type: 'checkbox' })`
+border: none;
+clip:rect(0 0 0 0);
+clippath: inset(50%);
+height: 0px;
+width: 1px;
+margin: -1px;
+overflow: hidden;
+padding: absolute;
+white-space: nowrap;
+`
+
+const StyledCheckbox = styled.div` 
+  display: inline-block;
+  width: 18px;
+  height: 18px;
+  background: ${props => props.checked ? '#4D1C39' : '#9A6A80'};
```

Comment:
Props inside style components is another thing I am very curious about and want to learn well. I like to see it in your project and learn from it:)